docker-compose.dev.yml

```diff
@@ -0,0 +1,88 @@
+services:
+  astarte-housekeeping:
+    build:
+      target: builder
+    command: >
+      mix run -e Elixir.Astarte.Housekeeping.ReleaseTasks.init_database || exit 1
+      mix run -e Elixir.Astarte.Housekeeping.ReleaseTasks.migrate || exit 1
+      mix run --no-halt
```

Comment:
Could it be the docker/docker-compose version?

Reply:
@noaccOS I found myself looking back at the docker-compose/yaml multiline talk and the situation is still different: the form of the classic multiline is what you were saying, but it has to become an element of the command array: 
```
command: - | set -e mix eval Elixir.Astarte.Housekeeping.ReleaseTasks.init_database mix eval Elixir.Astarte.Housekeeping.ReleaseTasks.migrate mix run --no-halt 
``` 
you can also use >, but you have to put everything in &&, since everything is folded in this case. My bad